modules/docs/Makefile.copyright

```diff
@@ -0,0 +1,22 @@
+COPYRIGHT_CMD ?= docker run --rm --volume `pwd`:$(COPYRIGHT_OUTPUT_DIR) osterman/copyright-header:latest
+COPYRIGHT_LICENSE ?= ASL2
+COPYRIGHT_HOLDER ?= Cloud Posse, LLC <hello@cloudposse.com>
+COPYRIGHT_YEAR ?= $(shell date +%Y)
+COPYRIGHT_SOFTWARE ?= Example App
```

Comment:
I changed my mind. Instead of setting a default value, leave it blank and instead use `assert-set` to ensure the value is set when `docs\:copyright-add` is called